new-pipeline/src/dist.js

```diff
@@ -3,7 +3,7 @@ var gFilterChangeTimeout = null;
 var gCurrentHistogramsList = null;
 var gCurrentDates = null;
 var gCurrentMinDate = null,
-  gCurrentMaxDate = null;
+    gCurrentMaxDate = null;
```

Comment:
There are a lot of whitespace changes in this file, that we don't need to change.
I get a syntax error in this file too when testing the changes locally.
Please drop the changes to this file.